.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2
+jobs:
+  centos7: &centos
+    docker:
+      - image: feedforce/ruby-rpm:7
+        user: builder
+    working_directory: /home/builder/rpmbuild
+    steps:
+      - run:
+          name: Build Ruby RPM
+          command: |
+            RUBY_X_Y_Z_VERSION=$(grep "%define \+rubyver" ./SPECS/ruby.spec | awk '{print $3}')
+            RUBY_X_Y_VERSION=$(echo $RUBY_X_Y_Z_VERSION | sed -e 's@\.[0-9]$@@')
+
+            cd ./SOURCES
+            curl -LO https://cache.ruby-lang.org/pub/ruby/$RUBY_X_Y_VERSION/ruby-$RUBY_X_Y_Z_VERSION.tar.gz
+
+            rpmbuild -ba ../SPECS/ruby.spec
+
+            mkdir /tmp/ruby-rpm
+            cp ../RPMS/x86_64/* /tmp/ruby-rpm
+            cp ../SRPMS/* /tmp/ruby-rpm
```

Comment:
This script is same as `script/build-ruby.sh`.
(I'd like to change output path from `/shared`. And, I'd like to extract script file from Docker image.)

Should I keep to use script file (like `script/build-ruby.sh`) instead of embedded script?

Reply:
No, I believe it's no problem.